### ISSUE TITLE
Unable to resolve modules css-loader after react update

### ISSUE THREAD

**Issue by wvanooijen92** [fritz-c/react-sortable-tree#217]
I just updated React to 16.2.0 and now my application fails.

`
Unable to resolve some modules:
  "!!../css-loader/index.js??ref--2-1!../postcss-loader/lib/index.js??ref--2-2!./styles.css" in /Users/user/project/adminv2/node_modules/react-sortable-tree/dist/main.js
(web.browser)
  "!!../node_modules/css-loader/index.js??ref--1-1!../node_modules/postcss-loader/lib/index.js??ref--1-2!../node_modules/sass-loader/lib/loader.js!./tree-node.scss" in
/Users/user/project/node_modules/react-sortable-tree/dist/main.js (web.browser)
  "!!../node_modules/css-loader/index.js??ref--1-1!../node_modules/postcss-loader/lib/index.js??ref--1-2!../node_modules/sass-loader/lib/loader.js!./node-renderer-default.scss" in
/Users/user/project/node_modules/react-sortable-tree/dist/main.js (web.browser)
  "!!../node_modules/css-loader/index.js??ref--1-1!../node_modules/postcss-loader/lib/index.js??ref--1-2!../node_modules/sass-loader/lib/loader.js!./placeholder-renderer-default.scss"
in /Users/user/project/node_modules/react-sortable-tree/dist/main.js (web.browser)
  "!!../node_modules/css-loader/index.js??ref--1-1!../node_modules/postcss-loader/lib/index.js??ref--1-2!../node_modules/sass-loader/lib/loader.js!./react-sortable-tree.scss" in
/Users/user/project/node_modules/react-sortable-tree/dist/main.js (web.browser)
`

Has anyone experienced something like this?

**Comment by orgpark** [comment]
i am wondering how this issue gets to be closed? would you please tell me how?

**Comment by buzztone** [comment]
The issue was closed by wvanooijen92 who originally posted the issue. You can see this clearly in the above info.

**Comment by wvanooijen92** [comment]
It's been a while since this issue so I am not completely sure what the solution was, but I am quite sure this had nothing to do with this package. I am using Meteor, which is using less instead of scss. I think the issue was resolved by adding the `fourseven:scss` meteor package. Sorry for the inconvenience!

**Comment by orgpark** [comment]
I'm also using Meteor and sadly fourseven:scss did not help. Really want to get rid of this warning. Someone please help

**Comment by wvanooijen92** [comment]
Yes you are right, today I checked out on the specific feature branch and it still gave me the same error. I have forked the package to see what it is. I noticed that styles are imported as an object and that these object are applied to the DOM as classNames. I have not seen this before, it looks similar to how [Aphrodite](https://github.com/Khan/aphrodite) works, but I see no such packages. I am definitely not a webpack expert (Meteor is zero config) so for me it is really hard to tell why the `css-loader` is breaking.

**Comment by fritz-c** [comment]
I found that part causing the issue was in some dead code inside the bundle. That used to be removed by UglifyJS, but then I had removed UglifyJS for another issue (see https://github.com/fritz-c/react-sortable-tree/issues/202 and https://github.com/fritz-c/react-image-lightbox/issues/88).
I've decided that the issue here takes priority over those, because at least the UglifyJS issue has a well-known workaround. See if `v1.5.4` works for you.
